index.html

```diff
@@ -6,7 +6,7 @@
     <script src="https://www.w3.org/Tools/respec/respec-w3c" async="" class="remove"></script>
     <script class="remove">
       var respecConfig = {
-          specStatus: "ED",
+          specStatus: "FPWD",
```

Comment:
Since we have `specStatus=FPWD` in the `echidna` file, we don't have to modify the config in the ED here.

Reply:
@xfq this isn't going to be published by echidna, since it's going to be published as a FPWD.

However, the ED shouldn't say that it's anything other than an ED.  Addison, you can change this to generate the flat file for publication, but you should commit it to GH like that.